src/cloudant/database.py

```diff
@@ -1098,3 +1101,128 @@ def get_query_result(self, selector, fields=None, raw_result=False,
+            search query.
+            Requires the index to have faceting enabled.
+        :param list drilldown:  Optional list of fields that each define a
+            pair of a field name and a value. This field can be used several
+            times.  The search will only match documents that have the given
+            value in the field name. It differs from using
+            ``query=fieldname:value` only in that the values are not analyzed.
+        :param str group_field: Optional string field by which to group
+            search matches.  Fields containing other data
+            (numbers, objects, arrays) can not be used.
+        :param int group_limit: Optional number with the maximum group count.
+            This field can only be used if ``group_field`` query parameter
+            is specified.
+        :param group_sort: Optional JSON field that defines the order of the
+            groups in a search using ``group_field``. The default sort order
+            is relevance. This field can have the same values as the sort field,
+            so single fields as well as arrays of fields are supported.
+        :param int limit: Optional number to limit the maximum count of the
+            returned documents. In case of a grouped search, this parameter
+            limits the number of documents per group.
+        :param query: A Lucene query in the form of ``name:value``.
+            If name is omitted, the special value ``default`` is used.
+        :param ranges: Optional JSON facet syntax that reuses the standard
+            Lucene syntax to return counts of results which fit into each
+            specified category. Inclusive range queries are denoted by brackets.
+            Exclusive range queries are denoted by curly brackets.
+            For example ``ranges={"price":{"cheap":"[0 TO 100]"}}`` has an
+            inclusive range of 0 to 100.
+            Requires the index to have faceting enabled.
+        :param sort: Optional JSON string of the form ``fieldname<type>`` for
+            ascending or ``-fieldname<type>`` for descending sort order.
+            Fieldname is the name of a string or number field and type is either
+            number or string or a JSON array of such strings. The type part is
+            optional and defaults to number.
+        :param str stale: Optional string to allow the results from a stale
+            index to be used. This makes the request return immediately, even
+            if the index has not been completely built yet.
+        :param list highlight_fields: Optional list of fields which should be
+            highlighted.
+        :param str highlight_pre_tag: Optional string inserted before the
+            highlighted word in the highlights output.  Defaults to ``<em>``.
+        :param str highlight_post_tag: Optional string inserted after the
+            highlighted word in the highlights output.  Defaults to ``<em>``.
```

Comment:
You're correct, it's invalid.  Result from a quick curl test:
```
{..., "rows":[...,"highlights":{"name":["<em>aardvark</em>"]}}]}
```
Fixed in 90213c1.